### ISSUE TITLE
port to cs2

### ISSUE THREAD

**Issue by suhailasrulsani-git** [Fguyexe/CalloutBot#1]
Hi there.
any plan to port this to cs2 ?
thank you

**Comment by OosterwijkJack** [comment]
No, I made this as a joke lol. Have you been actually using it or did you just find it recently?

**Comment by suhailasrulsani-git** [comment]
ive been using it like 3 years ago in csgo and then i stopped playing. it is now cs2
and i remembered it is so good.
